### PR TITLE
Add Memento v1

### PR DESCRIPTION
**Pull request by mjram0s**

### CODE REVIEW

File: src/handler.jl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Do not share a stream between processes
 # The token would be shared so putting would give InvalidSequenceTokenException a lot
-struct CloudWatchLogHandler{F<:Formatter} <: Handler{F, Union{}}
```
Comment (mjram0s) [inline src/handler.jl:3]: This was throwing `LoadError: LoadError: too many parameters for type`

Comment (iamed2) [reply src/handler.jl:3]: This changed between Memento 0.12 and 0.13. If you want to support both, you'll need to make this conditional. I would just drop Memento 0.12.

Comment (iamed2) [reply src/handler.jl:3]: This changed between Memento 0.12 and 0.13. If you want to support both, you'll need to make this conditional. I would just drop Memento 0.12.

File: Project.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name = "CloudWatchLogs"
 uuid = "c4c1e6a2-6bdd-52e0-b56d-1d4734724d2d"
-version = "1.1.3"
+version = "1.1.4"
```
Comment (iamed2) [inline Project.toml:3]:
```suggestion
version = "1.2.0"
```
Since we're now dropping a version of Memento

**Pull request merged**